skxray/correlation.py

```diff
@@ -219,8 +204,8 @@ def multi_tau_auto_corr(num_levels, num_bufs, labels, images):
     # ending time for the process
     end_time = time.time()
 
-    logger.info("Processing time for {0} images took {1} seconds."
-                "".format(n, (end_time - start_time)))
+    logger.info("One Time Correlation - Processing time for {0} images took"
```

Comment:
I'm fairly certain that using old-style string formatting for log messages is appropriate for performance reasons